skycoin-api/tools/base58.c

```diff
@@ -286,7 +286,7 @@ bool b58enc(char* b58, size_t* b58sz, const void* data, size_t binsz)
     for (j = 0; j < (ssize_t)size && !buf[j]; ++j)
         ;
 
-    if (*b58sz <= zcount + size - j) {
+    if (*b58sz < zcount + size - j) {
```

Comment:
Are you sure, that it should be in that way ? It is part of original trezor code